abe/resource_models/event_resources.py

```diff
@@ -133,6 +134,12 @@ def post(self):
         new_event = db.Event(**received_data)
         if new_event.labels == []:  # if no labels were given
             new_event.labels = ['unlabeled']
+        else:
+            for label in new_event.labels:
```

Comment:
There's some duplicated code between this and the next change. I would factor these out into a common function that you can call from each location. Then debugs or optimizations that are applied, only need to be applied to one location.

In this case the function could go in this same file. That's not a pattern that the rest of the app uses, but it's legitimate to place a function that's only used in one module, near where it's used. This lowers the activation energy required to apply DRY.